client-v2/src/services/capiQuery.js

```diff
@@ -28,7 +28,13 @@ type Element = ImageElement;
 type Article = {
   webTitle: string,
   webUrl: string,
-  elements?: Element[]
+  elements?: Element[],
+  fields?: {
+    trailText?: string
+  },
+  frontsMeta: {
```

Comment:
Does `tone` refer to the tone tag coming from capi? If this is the case, storing it in the an object called `frontsMeta` is a bit misleading as in the old tool (and I've been replicating this in the new tool) meta refers to overrides we make in the fronts tool.

Reply:
This is how they come back directly from CAPI and this is the type that represents the response. We should definitely transform them though at the point these get bought into a front (drag and drop or similar)

Reply:
Ah, I think I understand now. There is some default meta for articles stored in capi - we need to figure out how this works and what we should do with it.